aws-sdk-rails.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |spec|
   spec.license       = 'Apache-2.0'
 
   spec.require_paths = ['lib']
-  spec.files += Dir['lib/**/*', 'bin/*']
+  spec.files += Dir['lib/**/*', 'bin/*', 'app/**/*']
```

Comment:
assuming this is correct

Reply:
I don't know how this works, to be honest, this is the deepest I've gone with rails gems

Reply:
The fact that it's in app doesn't matter, I think you can move the file as is into lib and strip off app/controllers and it's still fine.

Reply:
I don't believe so due to autoloading. Based on another gem that add controller, this is OK https://github.com/ankane/ahoy/tree/master/app/controllers/ahoy